### PR TITLE
Update home-page format; Sign Out button; Correct router path names

### PR DESCRIPTION
**Pull request by relandboyle**
This pull request represents an improvement to the layout of the HomePage component. I would not say it's perfect, but it is significantly improved. Ideas or suggestions are welcome.

Added Sign Out button to the MainLayout component. This is particularly helpful when working on auth-related development where the developer must sign in repeatedly.

Also updated router path names to adhere to Linkta naming conventions document, and updated 'fine print' links on the public view components.

Fixes # N/A

## Type of change

- [x] Other 🗒️Enhancement

## How Can this be tested? Testing Plan to review this PR

- [x] What are the steps to test this change? | Sign in, sign out, navigate around, and consider whether everything feels okay and makes sense.

## Checklist:

- [x] I have performed a self-review of my own code.
- [x] My changes generate no new warnings.
- [x] I have followed the [**Naming Conventions Guide**](./docs/NAMING_CONVENTIONS_GUIDE.md) for my changes.
- [x] I have ensured that my pull request title is descriptive.

### CODE REVIEW

File: client/components/layout/MainLayout.tsx [hunk 1]
```diff
@@ -42,6 +42,19 @@ const MainLayout: React.FC = () => {
     matching && setDrawerOpen(false);
   }, [matching, setDrawerOpen]);
 
+  const auth = getAuth();
+  const handleSignout = () => {
+    signOut(auth)
```
Comment (kinskiwu) [inline client/components/layout/MainLayout.tsx]: Shall we use a try-catch block to be consistent with the rest of the codebase?

**Review by kinskiwu — APPROVED** [review]
Looks good to me. Thanks for the good work!

File: client/components/layout/MainLayout.tsx (hunk 1 above)
Comment (relandboyle) [reply client/components/layout/MainLayout.tsx]: @kinskiwu Done.

**Review by lomeintheory — APPROVED** [review]
This all looks great to me, but I noticed as I was navigating between different routes, I found myself clicking on the Linkta logo/text in our header bar to navigate to the home page. Probably not within the scope of this ticket or important to implement, but just a thought.

**Pull request merged**